### ISSUE TITLE
Low-sample size but displayed issue for race/ethnic dimension

### ISSUE THREAD

**Issue by MeaganLP** [CityOfPhiladelphia/community-health-explorer#44]
When 2: only an issue for the racial ethnic categories - when sample size needs to be interpret with caution, please include the same name as the label in the note, for example: "Interpret with caution, low sample size: Asian, non-Hispanic; white, non-Hispanic."
*Enhancement (I'll add this as a separate issue as well, so you can delete this issue once the first part is done)
